es6-classes/example-class.js

```diff
@@ -0,0 +1,18 @@
+class ExampleClass {
+  constructor(name) {
+    this.name = name;
+  }
```

Comment:
It didn't need anything inside of it, it could have been left empty.

Reply:
Ok. I see. So `exampleClass` is changed to a `class` but the code block can be left empty? Such as:

> class ExampleClass {

}